### ISSUE TITLE
Extract emoji ID from possibly mixed-character clipboard data...

### ISSUE THREAD

**Issue by kukabi** [tari-project/wallet-android#266]
...in _add recipient_ screen.

**Comment by Tas4tari** [comment]
@kukabi What is the best way in which to test this on Android 10?

**Comment by kukabi** [comment]
@Tas4tari I don't think it's possible to test it on Android 10 because of the clipboard permission issue.

**Comment by Tas4tari** [comment]
@kukabi Thank you, will try and test this on browser stack

**Comment by Tas4tari** [comment]
@kukabi @talmahaj I tested this on Browser-stack with an 
Android v8 device which tested correctly.
**Steps taken to test:**

1. I pasted an emoji ID to Telegram
2. I added characters to the emoji ID
3. I then copied that emoji ID from Telegram and pasted it into the Add recipient screen
4. The characters were stripped out and the emoji ID was successfully pasted

**Test proof:**
https://www.loom.com/share/6f6fb6dc05064cd2845ecc43d6ccdac3
